tests/functional/acceptance/15532-em-warnings/ExpressionWarningsOnLogicTest.php

```diff
@@ -53,7 +53,7 @@ public function testCheckInvalid()
         );
         try {
             self::$webDriver->get($url);
-            sleep(1);
+            sleep(5);
```

Comment:
???

Reply:
The test is failing. Making sure it wasn't a speed issue.